### PR TITLE
Add functionality to make bigcat proofreading projects WIP

### PR DESCRIPTION
**Pull request by constantinpape**
Still pending new storage of cell tables in the hdf5s.

Also:
What channels should we export for the training data?
Serum and marker?!
Corrected or uncorrected?

### CODE REVIEW

**Comment by lorenzocerrone** [comment]
> Serum and marker?!
yes, but I would suggest to have nuclei as well. They will help a lot when you have dim infected cell indistinguishable from the background.

> Corrected or uncorrected?
Do you mean barrel correction? I would prefer uncorrected but I don't have strong opinion.

**Comment by wolny** [comment]
I agree with @lorenzocerrone:
- no harm in taking all 3 channels as input to the network
- training with uncorrected images

**Comment by constantinpape** [comment]
> > Serum and marker?!
> 
> yes, but I would suggest to have nuclei as well. They will help a lot when you have dim infected cell indistinguishable from the background.
> 

Sounds good. For bigcat which channel should be the primary one in bigcat?
(I.e. the one that is overlayed with the segmentation)

> > Corrected or uncorrected?
> 
> Do you mean barrel correction? I would prefer uncorrected but I don't have strong opinion.

Yes, exactly. Using uncorrected is fine with me.

**Comment by wolny** [comment]
> Sounds good. For bigcat which channel should be the primary one in bigcat?
(I.e. the one that is overlayed with the segmentation)

it should be still the serum channel

**Comment by constantinpape** [comment]
@lorenzocerrone @wolny 
I have implemented your suggestions, will check if it works tomorrow once we have the new hdf5 table and then merge.

One more thing:
As ground-truth I am exporting a label image with values `0`: not infected, `1`: infected, `-1`: background. Let me know if you want to change the values here.
